README.md

```diff
@@ -144,41 +149,44 @@ How to Train Your Dragon 2 (2014)|7.9
 
 ## Available operations
 
-A summary of the currently available operations in **Astro SDK Python**.
-* `load_file`: load a given file into a SQL table
-* `transform`: applies a SQL select statement to a source table and saves the result to a destination table
-* `truncate`: remove all records from a SQL table
-* `run_raw_sql`: run any SQL statement without handling its output
-* `append`: insert rows from the source SQL table into the destination SQL table, if there are no conflicts
-* `merge`: insert rows from the source SQL table into the destination SQL table, depending on conflicts:
-  * ignore: do not add rows that already exist
-  * update: replace existing rows with new ones
-* `export_file`: export SQL table rows into a destination file
-* `dataframe`: export given SQL table into in-memory Pandas data-frame
+
+The following are some key functions available in the SDK:
+
+- `load_file`: load a given file into a SQL table
+- `transform`: applies a SQL select statement to a source table and saves the result to a destination table
+- `truncate`: remove all records from a SQL table
```

Comment:
We probably forgot to update this in the past, `truncate` is now `drop_table` and it actually drops the table - could you fix this as part of this PR?

Reply:
Can we link this section to readthedocs link - https://astro-sdk.readthedocs.io/en/latest/ operator section?